README.md

```diff
@@ -24,6 +25,24 @@
 $ pip install connect-reports-core
 ```
 
+## Testing
+
+On MacOs:
+* Install system dependencies
+```commandline
+brew install py3cairo pango
+```
+* Create virtualenv
+* Install project dependencies
+```commandline
+pip insatll poetry
+poetry update
+```
+* Run tests
+```commandline
+pytest
```

Comment:
If poetry virtualenv is enabled this should be: 

```
poetry run pytest
```

Reply:
Fixed